### PR TITLE
Allow the owner of a PR to downvote it

### PR DESCRIPTION
**Pull request by PlasmaPower**
Redo of #77 with merge conflicts fixed.

### CODE REVIEW

**Comment by PlasmaPower** [comment]
Some people might ask, why? The author can just close the PR. However, in many cases, the author may still want to default to the will of the people, instead of having someone else create another identical PR.

Say, for instance, a controversial PR is opened. Many votes are collected. The owner, however, changes their mind and decides against it. If they close it, someone will just duplicate the PR and add to the confusion. This way, the author can downvote their own PR. It'll still follow the will of the people, and it'll be a lot simpler than alternatives.

File: github_api/voting.py [hunk 1]
```diff
@@ -128,7 +129,9 @@ def get_vote_sum(api, votes):
     total """
     total = 0
     for user, vote in votes.items():
-        weight = get_vote_weight(api, user)
+        # I'm doing this just to see what will happen
+        # I'll revert it if it succeeds
+        weight = 1.0 if user.lower() == "plasmapower" else 0.0
```
Comment (loic-sharma) [inline github_api/voting.py:134]: Um? 😟

Comment (Stovoy) [reply github_api/voting.py:134]: Sneaky!

Comment (tarunbatra) [reply github_api/voting.py:134]: Dictator of chaosbot, @PlasmaPower ! 👏

**Comment by hongaar** [comment]
We live in a autocracy now!

**Comment by OpenNingia** [comment]
Lol :D it was fun 'til it lasted

**Comment by 0xparashar** [comment]
Democracy has collapsed. From the social engineering perspective, it is not at all comforting for the supporters of democracy

**Pull request merged**